album.go

```diff
@@ -0,0 +1,16 @@
+package spotify
+
+type Album struct {
+	Meta
+	AlbumType        AlbumType `json:"album_type"`
+	AvailableMarkets []string  `json:"available_markets"`
+	Images           []Image
+	Name             string
+}
+
+type AlbumType string
+
+const (
+	Single_T AlbumType = "single"
+	Album_T  AlbumType = "album"
```

Comment:
I'd say we don't need the `_T` here... I assume it's short for "type", but it's not good practice in Go, as far as I know.
```suggestion
	Single AlbumType = "single"
	Album  AlbumType = "album"
```

Reply:
The issue is that `Album` here is the name of a type as well, this was my way of getting around it. `AlbumType` as the const also doesn't work...really struggled here.

Reply:
Perhaps `AlbumTypeAlbum` and `AlbumTypeSingle` would work, I guess. A bit verbose, though.